doc/faq.rst

```diff
@@ -244,3 +243,23 @@ program: Insert the following instructions in your main script::
 
 You can find more default on the new start methods in the `multiprocessing
 documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods>`_.
+
+How do I set a random_state for an entire execution?
+----------------------------------------------------
+
+For testing and replicability, it is often important to have the entire execution
+controlled by a single seed for the pseudo-random number generator used in algorithms
+that have a randomized component. Scikit-learn does not use its own global random state;
+whenever a RandomState instance or an integer random seed is not provided as an argument,
+it relies on the numpy global random state, which can be set using :func:`numpy.random.seed`.
+For example, to set an execution's numpy global random state to 42, one could execute
+the following in his or her script::
+
+    import numpy as np
+    np.random.seed(42)
+
+However, a global random state is prone to modification by other code during execution. Thus, the only
+way to ensure replicability is to pass ``RandomState`` instances everywhere. In ``cross_val_score``
```

Comment:
Not if you provide an integer, or leave the default argument, for `cv`. In that case, cross_val_score will create a `StratifiedKFold` (or `KFold`) instance with default `random_state` argument. The resulting non-deterministic folds will cause `cross_val_score` to be non-deterministic.